compiler/luci/pass/src/QuantizeWeights.cpp

```diff
@@ -253,6 +255,12 @@ void QuantizeWeights::quantize_weights(luci::CircleConst *weights)
       return;
     }
 
+    for (uint32_t i = 0; i < quantparam->min.size(); i++)
+      if (range_check(quantparam->min[i], quantparam->max[i], output_type))
+        WARN(l) << weights->name()
+                << " cannot be represented accurately according to chosen quantization; " << i
+                << "-th channel's range is too great" << std::endl;
```

Comment:
Can you please open/close with curly brace for readability altough it's a single line?

Reply:
And others too